src/vcs/git.rs

```diff
@@ -788,6 +789,55 @@ impl<'a> Browser<'a> {
         self.repository.revision_branches(&commit.id())
     }
 
+    /// Get the [`Stats`] of the underlying [`Repository`].
+    ///
+    /// # Errors
+    ///
+    /// * [`error::Error::Git`]
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use radicle_surf::vcs::git::{Browser, Repository};
+    /// # use std::error::Error;
+    ///
+    /// # fn main() -> Result<(), Box<dyn Error>> {
+    /// let repo = Repository::new("./data/git-platinum")?;
+    /// let mut browser = Browser::new(&repo)?;
+    ///
+    /// let stats = browser.get_stats()?;
+    ///
+    /// // there should be 5 branches (local & remote)
+    /// assert_eq!(stats.branch_count, 5);
+    ///
+    /// // there should be 14 commits
+    /// assert_eq!(stats.commit_count, 14);
+    ///
+    /// // 4 contributors: Alexander Simmerl, Fintan Halpenny, FintanH, Rudolfs Ošiņš
+    /// assert_eq!(stats.contributor_count, 4);
+    ///
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn get_stats(&self) -> Result<Stats, Error> {
+        let history = self.repository.head()?;
+
+        let branch_count = self.list_branches(None)?.len();
+        let commit_count = history.iter().count();
```

Comment:
We can use `history.len()` here :)

Reply:
I get this error when I try using `len()` on `history`:

```
vcs.rs(30, 1): method `len` not found for this
git.rs(826, 36): method not found in `vcs::History<vcs::git::object::Commit>`
```

Reply:
Two options:
1) Add len to History
2) Access the NonEmpty underneath by doing `history.0.len()` 

I think 1) could make more sense, and it'll basically be doing 2). You'll have to implement `empty` too because clippy will suggest it.

Reply:
Sweet, will give this a go!

Reply:
Clippy didn't suggest `empty` 😱  should I do it anyway?

Reply:
Nevermind, clippy reared its head 😄  will add `is_empty`